src/sandboxpurpur/main.cpp

```diff
@@ -27,15 +23,15 @@ int main() {
 
     Object first;
 
-    auto s0 = event::Sound{ "jump.wav" };
-    auto s1 = event::Sound{ "shoot.wav" };
-    auto m0 = event::Message{ "best message" };
-
+    event::Message::connect(first);
+    event::Message::connect(first);
```

Comment:
Just a question to clarify: Calling `connect()` twice doesn't change anything, right?